client/state/google-apps-users/selectors.js

```diff
@@ -0,0 +1,21 @@
+import createSelector from 'lib/create-selector';
+
+function getGoogleAppsUsersState( state ) {
+	return state.googleAppsUsers;
+}
+
+function createGoogleAppsUsersSelector( fn ) {
```

Comment:
Interesting pattern here for generating memoized selectors with common cache breakers. I think I like it. @aduth any thoughts?